www/includes/easyparliament/templates/html/search/results.php

```diff
@@ -17,104 +18,104 @@
+                            <p class="search-result__description"><?= isset($result['speaker']) ? $result['speaker']['name'] . ': ' : '' ?><?= $result['extract'] ?></p>
+                        </div>
+                      <?php } ?>
 
                     <hr>
 
-                    <?php if ( $pagination_links ) { ?>
-                    <div class="search-result-pagination">
-                        <?php if ( isset($pagination_links['prev']) ) { ?>
-                        <a href="<?= $pagination_links['firstpage']['url'] ?>" title="First page">&lt;&lt;</a>
-                        <a href="<?= $pagination_links['prev']['url'] ?>" title="Previous page">&lt;</a>
-                        <?php }
-                        foreach ( $pagination_links['nums'] as $link ) { ?>
-                        <a href="<?= $link['url'] ?>"<?= $link['current'] ? ' class="search-result-pagination__current-page"' : '' ?>><?= $link['page'] ?></a>
-                        <?php }
-                        if ( isset($pagination_links['next']) ) { ?>
-                        <a href="<?= $pagination_links['next']['url'] ?>" title="Next page">&gt;</a>
-                        <a href="<?= $pagination_links['lastpage']['url'] ?>" title="Final page">&gt;&gt;</a>
-                        <?php } ?>
-                    </div>
-                    <?php } ?>
-                <?php } ?>
+                      <?php if ( $pagination_links ) { ?>
+                        <div class="search-result-pagination">
+                          <?php if ( isset($pagination_links['prev']) ) { ?>
+                            <a href="<?= $pagination_links['firstpage']['url'] ?>" title="First page">&lt;&lt;</a>
+                            <a href="<?= $pagination_links['prev']['url'] ?>" title="Previous page">&lt;</a>
+                          <?php } ?>
+                          <?php foreach ( $pagination_links['nums'] as $link ) { ?>
+                            <a href="<?= $link['url'] ?>"<?= $link['current'] ? ' class="search-result-pagination__current-page"' : '' ?>><?= $link['page'] ?></a>
+                          <?php } ?>
+                          <?php if ( isset($pagination_links['next']) ) { ?>
+                            <a href="<?= $pagination_links['next']['url'] ?>" title="Next page">&gt;</a>
+                            <a href="<?= $pagination_links['lastpage']['url'] ?>" title="Final page">&gt;&gt;</a>
+                          <?php } ?>
+                        </div>
+                      <?php } ?>
+                  <?php } # end of !isset($error) ?>
+                </div>
+
+                <?php include 'sidebar.php' ?>
             </div>
+          <?php } ?>
 
-            <?php include 'sidebar.php' ?>
-        </div>
-        <?php } ?>
+            <?php include 'form_options.php'; ?>
```

Comment:
As discussed with @dracos in IRC, this actually raises a bigger issue – if the sidebar can itself contain a form (the "Did you find what you were looking for?" mini survey) then we can't wrap the entire page in a form as this PR suggests.

The alternatives, therefore, are:

* Move the advanced search options div back up to the top of the page, before the search results, and end the form immediately after them. This means the sidebar will no longer be inside the main form, so the mini survey will work. But it forces non-javascript users to scroll or tab through all the advanced options before they get to the search results. (TBF, this is how it currently works!)
* Use two separate forms – one for the primary search input before the results section, and another for the advanced search options after the results. We'd need to duplicate the primary `#q` search input inside the advanced search form too. Then, when the javascript moves the advanced options div up above the results list, it would need to remove the extra input.